### ISSUE TITLE
Filter flagging landscapes as an explicit image. 

### ISSUE THREAD

**Issue by MPPebworth** [axodox/unpaint#31]
This is a really cool program you've built, and I prefer it over easy diffusion or others. I also appreciate the built-in NSFW filter. Some things are better not seen. However, I had one of my prompts flagged as an explicit image.
Positive: 
Jagged snow covered peaks rise from a snow covered forest, above a lake. A massive glacier descends down the mountain into the lake. realistic, crisp, Alps, Cascades
negative:
blurry, render, fuzzy

model: realistic vision onnx

Settings:
512X512
batching: 9
steps: 75
random seed: 1391526699
guidance strength: 7

**Comment by axodox** [comment]
Yes, it is known that the safety checker has some false positives. This is just the original one by StabilityAI, when I have more time I will look into finding better ones.
